guide/blueprints/custom-entities.md

```diff
@@ -46,14 +46,48 @@ before running `launch.command` relative to where that file is installed (or unp
 with the default `launch.command` being `./start.sh`.
 
 So if we create a file `/tmp/netcat-server.tgz` containing just `start.sh` in the root
-which consists of the two lines in the previous example,
+which contains the line `echo hello | nc -l 4321`, 
 we can instead write our example as: 
 
 {% highlight yaml %}
 {% readj example_yaml/vanilla-bash-netcat-file.yaml %}
 {% endhighlight %}
 
 
+#### Determining Successful Launch
+
+The default behaviour for `VanillaSoftwareProcess` is to run a command over ssh to do a 
+health check. The health check is done post-launch (repeating until it succeeds, before 
+then reporting that the entity has started).
+
+The default command will check if the pid written to `$PID_FILE` is running. This is why
```

Comment:
The default command used to carry out this health check will determine if the pid, written to `$PID_FILE` is running.